package.json

```diff
@@ -29,18 +29,18 @@
     "coveralls": "^2.11.15",
     "ember-a11y-testing": "0.1.6",
     "ember-ajax": "^2.0.1",
-    "ember-bootstrap": "1.0.0-alpha.3",
-    "ember-cli": "2.8.0",
-    "ember-cli-app-version": "^1.0.0",
+    "ember-bootstrap": "1.0.0-alpha.4",
+    "ember-cli": "2.11.1",
```

Comment:
Note to self: we'll want to pin ember to specific LTS releases; will evaluate pinning options for greenkeeper when I'm less tired.